OpenRA.Mods.Common/Traits/World/MapStartingUnits.cs

```diff
@@ -15,7 +15,7 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Used by SpawnMPUnits. Attach these to the world actor. You can have multiple variants by adding @suffixes.")]
-	public class MPStartUnitsInfo : TraitInfo<MPStartUnits>
+	public class MapStartingUnitsInfo : TraitInfo<MapStartingUnits>
```

Comment:
This trait really has nothing to do with maps, so this should be just `StartingUnits` to match `SpawnStartingUnits`.

Reply:
It certainly has something to do with maps, but I can see the point that this is not well worded since it doesn't spawn actors set by/depending on the map. Will change the name.